firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -101,12 +101,14 @@ class ToolbarMiddleware: FeatureFlaggable {
         var elements = [ToolbarActionState]()
         elements.append(ToolbarActionState(actionType: .tabs,
                                            iconName: StandardImageIdentifiers.Large.tab,
+                                           badgeImageName: StandardImageIdentifiers.Medium.privateModeCircleFillPurple,
```

Comment:
The `badgeImageName` should only be added to the `ToolbarActionState` if the badge needs to be displayed.